app/bundles/LeadBundle/EventListener/LeadSubscriber.php

```diff
@@ -529,18 +530,32 @@ protected function addTimelineDoNotContactEntries(Events\LeadTimelineEvent $even
 
                 if (!empty($row['channel'])) {
                     if ($channelModel = $this->getChannelModel($row['channel'])) {
+                        if ($channelModel instanceof ChannelTimelineInterface) {
```

Comment:
I see you're doing a check against `ChannelTimelineInterface` to see if the model implements it, but you didn't actually implement that interface on the appropriate classes (`Mautic\LeadBundle\EventListener\LeadSubscriber` and `Mautic\PageBundle\EventListener\LeadSubscriber`)